src/hubdata/app.py

```diff
@@ -1,32 +1,62 @@
+import pathlib
+
+import click
 import structlog
-from rich.console import Console
+from rich.console import Console, Group
 from rich.panel import Panel
 
+from hubdata import HubConfig, create_hub_schema
 from hubdata.logging import setup_logging
 
 setup_logging()
 logger = structlog.get_logger()
 
 
-def main():
-    """hubdata starting point."""
-    logger.info('starting hubdata...')
+@click.group()
+def cli():
+    pass
+
 
+@cli.command(name='schema')
+@click.argument('hub_dir', type=click.Path(file_okay=False, exists=True, path_type=pathlib.Path))
+def print_schema(hub_dir):
+    """
+    A subcommand that prints the schema for `hub_dir`.
+    """
+    hub_config = HubConfig(hub_dir)
+    schema = create_hub_schema(hub_config.tasks)
+    rounds = hub_config.tasks['rounds']
```

Comment:
Maybe a next idea, is to put the `round_id` tag as an input:
- if the round has `"round_id_from_variable": true`, extract the value from the variable instead of returning the variable for example `"2021-03-07"` instead of `"origin_date"`
- if the round has `"round_id_from_variable": false`, extract the value from the `"round_id"` (as it is currently)

Does that make sense? It will make the output easier to understand I think